spaceship/config.py

```diff
@@ -13,3 +13,6 @@ class Config:
   MYSQL_USERNAME = os.environ.get('MYSQL_USERNAME', 'spaceship-app')
   MYSQL_PASSWORD = os.environ.get('MYSQL_PASSWORD', 'aa7925b6f7b')
   MYSQL_DB = os.environ.get('MYSQL_DB', 'spaceship')
+
+  GOOGLE_CLIENT_ID = os.environ.get('GOOGLE_CLIENT_ID', '700492634886-qbhv3gss1a59lm5p93gr7plo872auaba.apps.googleusercontent.com')
```

Comment:
we should probably treat this cred the same as the sendgrid key, where if you want google oauth to work locally you can pull the secret off the k8s cluster. i can work on that.

Reply:
ack, i'll merge this first, since it seems harmless and least likely to mess up others' workflows for now

Reply:
tracked in issue #53